shell.nix

```diff
@@ -1,6 +1,6 @@
 (import ./default.nix).shellFor {
   tools = {
-    cabal = "3.2.0.0";
+    cabal = "latest";
```

Comment:
I find it's usually best to use `latest` here so you get a cached version of `cabal`.